tests/test_qfit_protein.py

```diff
@@ -32,10 +32,9 @@ class TestQFitProtein:
     def mock_main(self):
         # Prepare args
         args = [
-            "../example/3K0N.mtz",  # mapfile, using relative directory from tests/
-            "../example/3K0N.pdb",  # structurefile, using relative directory from tests/
-            "-l", "2FOFCWT,PH2FOFCWT",
-            "--directory", f"{os.environ['QFIT_OUTPUT_DIR']}",
+            "./example/3K0N.mtz",  # mapfile, using relative directory from tests/
+            "./example/3K0N.pdb",  # structurefile, using relative directory from tests/
+            "-l", "2FOFCWT,PH2FOFCWT"
```

Comment:
Ditto with the trailing comma.
```suggestion
            "-l", "2FOFCWT,PH2FOFCWT",
```